romancal/regtest/test_hlp_pipeline.py

```diff
@@ -0,0 +1,93 @@
+""" Roman tests for the High Level Pipeline """
+
+import pytest
+import roman_datamodels as rdm
+from metrics_logger.decorators import metrics_logger
+
+from romancal.pipeline.highlevel_pipeline import HighLevelPipeline
+
+from .regtestdata import compare_asdf
+
+
+def passfail(bool_expr):
+    """set pass fail"""
+    if bool_expr:
+        return "Pass"
+    return "Fail"
+
+
+@pytest.mark.bigdata
+@pytest.mark.soctests
+@metrics_logger("DMS356")
+def test_level3_hlp_pipeline(rtdata, ignore_asdf_paths):
+    """Tests for level 3 processing requirements DMS356"""
+
+    cal_files = [
+        "WFI/image/r0000101001001001001_01101_0001_WFI01_cal.asdf",
+        "WFI/image/r0000101001001001001_01101_0002_WFI01_cal.asdf",
+        "WFI/image/r0000101001001001001_01101_0003_WFI01_cal.asdf",
```

Comment:
Just remarking: these are three existing L1 files that we make, and three existing cal file regtests that we used derived from them.
https://github.com/spacetelescope/romanisim/blob/main/scripts/romancal_make_regtest_l1s.sh
https://github.com/spacetelescope/romancal/blob/main/romancal/scripts/make_regtestdata.sh
I had previously thought these were new and wanted to make sure we also updated these scripts accordingly.

01, 03, and 04 are at the same positions, and 02 has a slight dither.  We could consider placing 03 and 04 at small dithers if we wanted a bit more diversity.

Reply:
For the tests these need to be generated from the uncal products via the elp.